tests/cypress/e2e/benefit.cy.js

```diff
@@ -0,0 +1,58 @@
+describe('Vacation requests', () => {
+
+  beforeEach(() => {
+    cy.refreshDatabase()
+    cy.seed('DemoSeeder')
+
+    cy.php (`Toby\\Eloquent\\Models\\User::first();`)
+      .then(user=> {
+        cy.login({email: user.email})
+      })
+  })
+
+  it('try to create benefit, add it to a user and check if calculation are correct', () => {
+    cy.visit('/benefits');
+
+    cy.attr('create-benefit')
+      .click()
+    
+    cy.get('#name')
+      .type('gym')
+
+    cy.attr('save-benefit')
```

Comment:
Better naming for `data-cy` attribute - what it `save-benefit`? It's button or link or what? I think better name will be `save-benefit-button`. This applies to all names in `data-cy` tags.